build-tools/src/test/groovy/org/elasticsearch/gradle/testclusters/SslTrustResolverSpec.groovy

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.gradle.testclusters
+
+import spock.lang.Specification
+
+import java.nio.file.Paths
+import java.security.cert.Certificate
+import java.security.cert.CertificateException
+import java.security.cert.CertificateFactory
+import javax.net.ssl.TrustManager
+import javax.net.ssl.X509TrustManager
+
+class SslTrustResolverSpec extends Specification {
```

Comment:
@mark-vieira I am a long way from being an export on Spock, so if you want to critique my spec, feel free to do so.

Reply:
Looks great other than some unnecessary `;`s 😉